k-distribution/src/main/scripts/lib/setenv

```diff
@@ -9,7 +9,7 @@ else
 fi
 K_LIB_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
-export PATH="$K_LIB_DIR/../../bin:$PATH"
+export PATH="$K_LIB_DIR/../../bin:$K_LIB_DIR/bin:$PATH"
```

Comment:
since you are including the lib/bin path, it might be better to just patch ng there, i.e.

```
preFixup = lib.optionalString (!stdenv.isDarwin) '' 
  patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" "$out/lib/bin/ng" 
''; 
```